src/components/InventoryV2/view/InventorySeedlingsTable.tsx

```diff
@@ -414,7 +428,7 @@ export default function InventorySeedlingsTable(props: InventorySeedlingsTablePr
           <Box marginTop={theme.spacing(2)}>
             <Table
               id='inventory-seedlings-table'
```

Comment:
Make this `{`inventory-seedlings-table-${origin.toLowerCase()}`}` so we have unique table ids by use-case.
The id is used as the preference name to store table column ordering, if user shuffles the columns.